conftest.py

```diff
@@ -63,6 +63,17 @@ def container_git_clone(request: SubRequest, tmp_path):
             f"{container_fixture.container_id}:/{git_repo_build.repo_name}",
         ]
     )
+    # fix file permissions for the git copied git repo
+    check_output(
+        [
+            runtime.runner_binary,
+            "exec",
+            container_fixture.container_id,
+            "/bin/sh",
+            "-c",
+            f"chown --recursive $(id -u):$(id -g) {git_repo_build.repo_name}",
```

Comment:
there gotta be a better way than this... is the issue that we'r enot setting a home directory on the container?

Reply:
No the issue is that `docker cp ` operates in archive mode by default preserving the uid & gid of the git repository. So the cloned repo has everything owned by uid 1000 (well, on my box, but generally != 0) and the process in the container is run under uid 0. And with that anything invoking git on that repo fails due to an ownership missmatch.